### ISSUE TITLE
NMMNH Events

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/data-migration#146]
@Nicole-Ridgwell-NMMNHS some of the records in Locality 1 have an odd date and I need to know how to treat them. The cdate is **5/72005**. There are also records dated 5/2005, so this could be a typo and perhaps the 5/72005 dates should be "5/2005" or the odd structure might indicate "May through July of 2005" or it could be a typo of "5/7/2005". Based on all of this, you should probably review all of the records with 5/2005 and 5/72005 in cdate.

Thanks!

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
Here are the collected dates that are in our log:

45555     5/7/05
45556-63     5/6/2005
the 5/2005 ones are correct

**Comment by Jegelewicz** [comment]
45564 also has the 5/72005 date. What should I do with it?

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
sorry, yes 45564 should also be 5/6/2005. I think at some point it was in the database correctly and someone messed it up because the old access-generated tags with the specimens have the date correctly.

**Comment by Jegelewicz** [comment]
@Nicole-Ridgwell-NMMNHS I am wondering if we should treat specimens with "no date" of collection as follows:

Verbatim date = no date
Began date = datefound from locality (it shouldn't have been collected before the site was found)
Ended date = day of upload (we know it was collected before today, whatever today is at the moment)

**Comment by dustymc** [comment]
>no date

http://handbook.arctosdb.org/documentation/collecting-event.html#dates

Something like 'before {today}' is much less likely to be confusing in the future.

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
how about verbatim date = "no date, began date is locality datefound, end date is day of upload"
A bit wordy, but clearly explains how we came to those began and end dates. Eventually we could pull a list of all those specimens, find the date date they were entered in our specimen ledger, and constrain the end date better.

**Comment by Jegelewicz** [comment]
> no date, began date is locality datefound, end date is day of upload

exceeds limit of 60 characters, so using

> no date, began is locality datefound, end is day of upload
